### ISSUE TITLE
Bot Prevention

### ISSUE THREAD

**Issue by datenbrei** [Human-Connection/Human-Connection#280]
As a network responsible I'd like to keep out bot's from the network in order to enshure, that just physical persons are posting content to the network

**Comment by Mastercuber** [comment]
In ActivityPub you can differ if a user is a physical one, where the actor object has the type Person, or if it's a Service, Applikation etc. 
The only thing here is, that a bot could also be hidden as a Person. So the differentiation through the type is not always correct.

**Comment by mattwr18** [comment]
I think this is still relevant as we'd like some sort of captcha, no?

**Comment by datenbrei** [comment]
@roschaefer @Mastercuber @mattwr18  For ActivityPub content that may be federated to our instance we possibly need to disable the view of the "fererated timeline", because we can not moderate this. A per-person-view of subscribed authors/articles from other instances may be ok, though.

We will have to discuss this in more depth. Bot prevention may be a topic not 100% dependent on ActivityPub. I will enrich the issue's description.

**Comment by sushidave** [comment]
Obviously, there is a bunch of spam accounts in the network now, like these:

![hc-bot-prevention-spammed-accounts](https://user-images.githubusercontent.com/17728384/74883412-2ecfc680-5371-11ea-92df-9c27b9165c7d.jpg)
